tembo-operator/README.md

```diff
@@ -1,34 +1,42 @@
 # Tembo Operator
 
-## Requirements
+The tembo operator is responsible for reconciling the differences between desired and actual cluster state. _Pad this out with around three sentences of explanation of the purpose of this component_.
 
-- A Kubernetes cluster: [kind](https://github.com/kubernetes-sigs/kind)
-- [rust](https://www.rust-lang.org/)
-- [just](https://github.com/casey/just)
-- [helm](https://helm.sh/)
-- Opentelemetry collector (**optional**)
```

Comment:
Most of these are somewhat necessary, but I found the telemetry stuff to get in the way of comprehension as a first-time reader.